project1/css/index.css

```diff
@@ -62,3 +62,102 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+nav {
+	padding-top:2%;
+	width:100%;
+	margin-bottom: 1px;
+	text-align:center;	
+}
+
+nav li {
+	padding-left:7%;
+	float: left
+}
+
+nav li a {
+	text-decoration:none;
+	color:grey;
+}
+
+.logo{
+	float:right;
+	margin-right:50px;
+}
+.cta{
```

Comment:
Since you have a line space separating previous code blocks, try to keep consistency.

Also, space between `.cta` and `{`